sw/device/lib/dif/BUILD

```diff
@@ -105,11 +105,10 @@ cc_library(
 cc_test(
     name = "uart_unittest",
     srcs = [
-        "autogen/dif_uart_autogen.inc",
+        "autogen/dif_uart_autogen.h",
```

Comment:
Also: `autogen/dif_uart_autogen_unittest.cc` and similar for the other tests below.